tiatoolbox/models/engine/semantic_segmentor.py

```diff
@@ -351,6 +351,35 @@ def __getitem__(self, idx: int):
 class SemanticSegmentor:
     """Pixel-wise segmentation predictor.
 
+    The tiatoolbox model should produce the following results on the BCSS dataset
+    using fcn_resnet50_unet-bcss.
+
+    .. list-table:: Semantic segmentation performance on the BCSS dataset
```

Comment:
```suggestion
    .. list-table:: Semantic segmentation performance on the BCSS dataset.
```

Reply:
Sentence follows on next line, so we shouldn't be using a full stop.